bots/ts/softcore/Makefile

```
@@ -0,0 +1,206 @@
+
+PLAYER1_BOT := my-core-bot/index.ts
+PLAYER2_BOT := gridmaster/index.ts
+
+PLAYER1_DIR = $(patsubst %/,%,$(dir $(PLAYER1_BOT)))
+PLAYER2_DIR = $(patsubst %/,%,$(dir $(PLAYER2_BOT)))
+
+PLAYER1_BOT_NAME = $(notdir $(PLAYER1_BOT))
+PLAYER2_BOT_NAME = $(notdir $(PLAYER2_BOT))
+
+
+run: stop check_update_configs
```

checkmake (0.2.2)

[warning] Target body for "run" exceeds allowed length of 5 (9).

(maxbodylength)